src/models/AccessRights.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * LINKURIOUS CONFIDENTIAL
+ * Copyright Linkurious SAS 2012 - 2019
+ *
+ * - Created on 2019-09-12.
+ */
+
+// TS2019-DONE
+
+import {IDataSourceParams} from './DataSource';
+
+export interface ISetAccessRightsParams extends IDataSourceParams {
+  groupId: number;
+  type: string;
+  targetType: string;
+  targetItemType?: string;
```

Comment:
what is this for?

Reply:
to set the itemType (e.g. 'COMPANY') to which an access right is referring too when targetType is 'nodePropertyKey'.

Reply:
this this should be an union type as well, like `"nodeCategory", "edgeType", "nodePropertyKey", "edgePropertyKey"`

Reply:
that's targetType and not targetItemType.
Calling it targetLabel would have been clearer, but we prefered to rename "label" to "itemType".

Reply:
Ok, now I understand, so it's `targetType` that should be union type.

BTW adding a example of value for each field that is not a union type in an inline comment in the interface could be helpful to get an idea of that these fields are.